source/basic.tex

```diff
@@ -6290,7 +6290,7 @@
 \begin{note}
 This states that operations on
 ordinary objects are not visibly reordered. This is not actually detectable
-without data races, but it is necessary to ensure that data races, as defined
+without data races, but needed to ensure that data races, as defined
```

Comment:
```suggestion
without data races, but is needed to ensure that data races, as defined
```

I think this reads a little more naturally to me.

Reply:
Agreed, esp. given since the sentence goes on for quite a bit after this.